.ci/tweet-releasenotes.sh

```diff
@@ -32,17 +32,28 @@ CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
 echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
 
 cd .ci-temp/checkstyle
-LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
-                       | jq ".tag_name")
-echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
+
+curl https://api.github.com/repos/checkstyle/checkstyle/releases -o /var/tmp/cs-releases.json
```

Comment:
add auth token

Reply:
Changes Done

Reply:
I am not sure if adding an `AUTH TOKEN` is a good idea. 

My API request never got rejected without `AUTH TOKEN` but with `AUTH TOKEN`, sometimes my expected outcome is not the same.

Example:
![Screenshot from 2022-08-17 19-19-39](https://user-images.githubusercontent.com/71710042/185151962-fb9cfd6a-e009-4e6b-ae14-d026e4bafcb8.png)


Code, if want to test:

```
 curl \
 https://api.github.com/repos/checkstyle/checkstyle/releases \
 -o /var/tmp/cs-releases.json

TARGET_RELEASE_NUM=10.1
TARGET_RELEASE_INDEX=$(cat /var/tmp/cs-releases.json | \
    jq '[.[].tag_name] | to_entries | .[] | select(.value=="checkstyle-'$TARGET_RELEASE_NUM'") | .key')
echo "$TARGET_RELEASE_INDEX"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  115k  100  115k    0     0   364k      0 --:--:-- --:--:-- --:--:--  367k
```
OUTPUT: `4`

```
 curl \
 https://api.github.com/repos/checkstyle/checkstyle/releases \
 -H "Authorization: token ghp_..." \
 -o /var/tmp/cs-releases.json

TARGET_RELEASE_NUM=10.1
TARGET_RELEASE_INDEX=$(cat /var/tmp/cs-releases.json | \
    jq '[.[].tag_name] | to_entries | .[] | select(.value=="checkstyle-'$TARGET_RELEASE_NUM'") | .key')
echo "$TARGET_RELEASE_INDEX"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    90  100    90    0     0    158      0 --:--:-- --:--:-- --:--:--   158
```

ERROR: `jq: error (at <stdin>:4): Cannot index string with string "tag_name"`

Reply:
>Cannot index string with string "tag_name"

it means that response was not a json, please execute command few times without `| jq` to see that response you got and please share it here.

Reply:
Works fine now. The error was coming from the quotes at `select(.value=="checkstyle-'$TARGET_RELEASE_NUM'"`